daprdocs/layouts/shortcodes/dapr-latest-version.html

```diff
@@ -0,0 +1,5 @@
+{{ if .Get "version" }}
+1.4
+{{ else }}
+1.4.0
+{{ end }}
```

Comment:
Maybe a nit, but I would prefer the implementation to be more explicit to deal with when the shortcode specifies either version="short" or version="long" to avoid someone in the future using something like ``{{% dapr-latest-version version="long" %}}`` and getting unexpected results. The ``else`` to default to the long version makes sense though.

Reply:
Done.